_posts/2021-10-01-selection-sort.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Selection Sort - A simple sorting algorithm"
+description: "This blog post is about Selection Sort, a simple sorting algorithm. Here, we will talk more about selection sort, how it works, and finally look at its pseudo-code for its implementation."
+keywords: 
+    - Mentro
+    - Selection Sort
+    - Sorting Algorithm
+category: dsa
+layout: post
```

Comment:
The layout is not required.

Reply:
Ok. Will remove that and send PR once again.